### DISCUSSION TITLE
Raspberry Farming Sleep Problem

### DISCUSSION THREAD

**Discussion by ogunkakisim** [Chia-Network/chia-blockchain#8162] [Support]
Hello friends, I want to use Raspberry PI 4 and Raspberry OS 64-Bit for Chia Farm, but; HDDs go to sleep after a certain period of time while farming, is there a way to prevent this? All disks are WD External Mybook, my USB hub is TP Link Uh 720. Thanks

**Comment by brightways021** [comment]
Hi, By default many external enclosures spin down the disk or put them into sleep after 10 mins so with this little script you can make sure it does not happen:
`sudo nano /usr/bin/keepalive.sh`

Add:

`#!/bin/bash
DATE=date
echo $DATE > /mnt/hdd1/keepalive.io
echo $DATE > /mnt/hdd2/keepalive.io
sync`

Then schedule it with crontab to run it every 9 minutes:

`crontab  -e`
`# then add this to the cron file
*/9 * * * * /usr/bin/keepalive.sh`

for ubuntu. tnx

**Reply by ogunkakisim** [reply]
Thank you very much for your help and reply. Is my images correct?
![Ekran görüntüsü 2021-08-28 001006](https://user-images.githubusercontent.com/57409405/131188998-17bec0bf-82c4-4819-9d70-9ccc97fae89a.png)
![Ekran görüntüsü 2021-08-28 001038](https://user-images.githubusercontent.com/57409405/131189002-dd68520a-f950-4cd5-9105-d9ed8b3f2f2e.png)
@brightways021

**Reply by loppefaaret** [reply]
in the crontab, you need to remove the leading `#`, otherwise the entire line will be ignored

**Reply by ogunkakisim** [reply]
thank you very much. i revised.

**Comment by BrandtH22** [comment]
@ogunkakisim , based on your previous message it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
